import

```diff
@@ -315,29 +386,142 @@ var importPerson = function(obj) {
     });
     req.write(JSON.stringify(person));
     req.end();
-  }, stats.rows * wait);
+  }, stats.rows * options.wait);
+};
+
+var importPlace = function(obj) {
+  var config = supportedTypes[options.recordType],
+      place = {
+        imported_date: new Date().toISOString(),
+        parent: obj.parent || options.placeUUID
+      };
+  Object.keys(obj).forEach(function(key) {
+    if (key === 'uuid') {
+      place._id = obj[key];
+    } else if (key.match(/^contact\.\w+/) || key.match(/^parent\.\w+/)) {
+      var props = key.split('.');
+      if (!place[props[0]]) {
+        place[props[0]] = {};
+      }
+      if (props[1] === 'uuid') {
+        place[props[0]]._id = obj[key];
+      } else {
+        place[props[0]][props[1]] = obj[key];
+      }
+    } else {
+      place[key] = obj[key];
+    }
+  });
+  place.type = config.doc_type;
+  setTimeout(function() {
```

Comment:
I don't think this setTimeout actually does anything. If I'm reading it right all the objects will get read in as quickly as possible then `options.wait` ms later they'll all get submitted. I think what you're trying to do is queue the requests so they don't all happen at once but that needs a callback to ensure only 1 is being run at a time. Consider using https://github.com/caolan/async/blob/v1.5.2/README.md#queue or similar?